app/views/hyrax/dashboard/collections/edit.html.erb

```diff
@@ -4,33 +4,9 @@
   <h1><span class="fa fa-edit"></span><%= t('.header', type_title: @collection.collection_type.title, title: @form.title.first) %></h1>
 <% end %>
 
-<% unless has_collection_search_parameters? %>
-  <div class="row">
-    <div class="col-md-12">
-      <%= render 'flash_msg' %>
-      <%= render 'form' %>
-    </div>
+<div class="row">
+  <div class="col-md-12">
+    <%= render 'flash_msg' %>
+    <%= render 'form' %>
   </div>
-
-  <div class="row">
-    <div class="col-md-12">
-      <div class="panel panel-default">
-        <div class="panel-body">
-          <%= render 'edit_actions' %>
-
-          <div class="row m-t-3">
```

Comment:
I believe this is the only place in the app this class is used, so it can be removed from the stylesheet.